main.py

```diff
@@ -410,7 +422,25 @@ def main():
         args.read_only = False
     if not hasattr(args, "retry_sec"):
         args.retry_sec = 60
-
+    if not hasattr(args, "t_token") or args.t_token is None or not hasattr(args, "t_id") or args.t_id is None:
+        try:
```

Comment:
du könntest dir das sparen indem du direkt json.load(f) verwendest
Beispiel bei `kontaktdaten.py` - `get_kontaktdaten` - Zeile 36 

Zudem vielleicht hier etwas vereinfacht ein Vorschlag :)
```suggestion
        try:
            with open('data/telegram.json') as f:
                telegram_json = json.load(f)

            args.t_token=telegram_json["token"]
            args.t_id=telegram_json["chatid"]
        except json.JSONDecodeError:
            print("Error: Fehler beim parsen der telegram daten")

        except KeyError:
            print(f"Error: telegram.json muss 'token' und 'chatid' beinhalten")

        else:
            args.t_token=None
            args.t_id=None
```

Reply:
siehe 6be5b7d